cmd/snap-bootstrap/initramfs_systemd_mount.go

```diff
@@ -144,32 +143,21 @@ func doSystemdMountImpl(what, where string, opts *systemdMountOptions) error {
 		args = append(args, "--options="+strings.Join(options, ","))
 	}
 
-	// note that we do not currently parse any output from systemd-mount, but if
-	// we ever do, take special care surrounding the debug output that systemd
-	// outputs with the "debug" kernel command line present (or equivalently the
-	// SYSTEMD_LOG_LEVEL=debug env var) which will add lots of additional output
-	// to stderr from systemd commands
-	out, err := exec.Command("systemd-mount", args...).CombinedOutput()
-	if err != nil {
-		return osutil.OutputErr(out, err)
-	}
-
 	// if it should survive pivot_root() then we need to add overrides for this
 	// unit to /run/systemd units
 	if !opts.Ephemeral {
-		// to survive the pivot_root, we need to make the mount units depend on
```

Comment:
does the order in which we call systemd-mount and write these unit files matter?

Reply:
I do not think this should matter.

Reply:
ack thanks for confirming